Samples/ExternalCamera/ExternalCameraSample/ViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ViewController.swift
+//  ExternalCameraSample
+//
+//  Copyright © 2019 Mapbox. All rights reserved.
+//
+
+import UIKit
+import MapboxVision
+
+class ViewController: UIViewController, VisionManagerDelegate {
+    
+    private var fileVideoSource: FileVideoSource!
+    private var visionMananger: VisionManager!
+    private let visionViewController = VisionPresentationViewController()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        addChild(visionViewController)
+        view.addSubview(visionViewController.view)
```

Comment:
It's also good thing to add `visionViewController.didMove(toParentViewController: self)` after adding view as a subview.

Reply:
Also aren't constraints needed here?